readme.md

```diff
@@ -6,54 +6,122 @@
 
 NcEngine is a 3D game engine written in C++20 targeting Windows. It is actively being developed but is still in very early stages. Features are in varying stages of completeness.
 
-## Links
 -----------
+Getting started:
+* [Requirements](#requirements)
+* [Targets](#targets)
 * [Building](#building)
-    * [Configurations](#configurations)
-    * [Definitions](#definitions)
+    * [Command Line](#command-line)
+    * [Visual Studio](#visual-studio)
+* [Configuration and Presets](#configuration)
+* [Definitions](#definitions)
+
+More information:
 * [Overview](docs/Overview.md)
 * [Creating a Project](docs/CreatingAProject.md)
 * [NcEngine Components](docs/EngineComponents.md)
 
 
-## Building
------------
-Current limitations:
-* NcEngine depends on Windows. There will likely be cross-platform support in the future, but it is not actively being pursued.
-* There are a few places where gcc specific code hasn't been fixed. This unfortunately means that mingw is the only currently supported compiler. This is planned to be fixed in the near future.
-
-There are three CMake projects in the repository
-* The engine, located in the 'nc' directory, is built as a static library and is output to 'nc/lib'.
-* An editor, located in the 'editor' directory, is output to the repository directory as 'NcEditor.exe'.
-* A sample project, located in the 'project' directory, is output to the repository directory as 'Sample.exe'.
+### Requirements:
```

Comment:
The ':" are inconsistent in the headers - I don't have a preference for ":" or none however, either way.